skeletons/chrome/content-script.js

```diff
@@ -89,35 +89,34 @@
   /**
    * Send the iframes to EmberInspector so that it can run
    * EmberDebug in the context of the iframe.
+   *
+   * @param {DOMEvent} event
    */
 
-  function sendFrameToInspector(url) {
-    chrome.extension.sendMessage({type: 'iframes', urls: [url]});
+  function sendFrameToInspector(event) {
+    chrome.extension.sendMessage({type: 'iframes', urls: [event.target.src]});
+  }
+
+  function setupOnLoadListenersForIFrames() {
+    // get all current frames and attach onLoad event
+    var iframes = document.getElementsByTagName('iframe');
+    for (var i = 0, l = iframes.length; i < l; i ++) {
+      var iframe = iframes[i];
+      if (!('__inspector__on__load_listener' in iframe)) {
```

Comment:
I am not sure about this... Maybe removeEventListener would still be better or some other way?